resources/language/resource.language.es_es/strings.po

```diff
@@ -942,4 +946,4 @@ msgstr "Notificaciones de Up Next (ver vídeo siguiente)"
 
 msgctxt "#30239"
 msgid "Color of supplemental plot info"
-msgstr "Color de la info. argumental complementaria"
+msgstr "Color de la información extra para argumento"
```

Comment:
I had several doubts with this one. But I think it's not the 'color of extra information for plot', but 'color for plot' itself. The source string (english) is not very clear (@CastagnaIT , please review this one), so my new proposal is:
English: Text color for plot information
Spanish: Color del texto para el argumento

Reply:
difficult to explain in words, but it is not the color of the full Plot, in practice other information are added to the Plot, it are called "supplemental"
Here on Kodi there is no specific field for this Netflix information, so i had to unify this text with the Plot, but with a different color in order to distinguish.

Example:
Rectangle Blue: is the full Plot text
Rectangle Yellow: is the "supplemental" info text, added to the plot
![image](https://user-images.githubusercontent.com/3257156/79197888-d8dc4400-7e32-11ea-8523-830652dcf53d.png)

Reply:
Ok, then @roliverosc is right, it's an additional information for the plot. I think it's ok his proposal, but in order to be more natural, I would prefer:
_Color de la información extra para **el** argumento_
OR
_Color de la información extra **del** argumento_
And if size is an issue:
_Color de la **info.** extra **del** argumento_

Reply:
I think we can save 'del' makes more sense 'para'